asset-manager/README.md

```diff
@@ -200,81 +197,74 @@ To successfully complete this workshop, you need the following:
-
-In VSCode, configure runtime arguments to enable the proposed API:
-```json
-  "enable-proposed-api": ["Microsoft.migrate-java-to-azure"],
-```
-1. Press **Ctrl+Shift+P** and select **Preferences: Configure Runtime Arguments**.
-2. Add the above JSON snippet into the editor and save.
-3. Restart VSCode.
+## Install GitHub Copilot App Modernization for Java
 
+In VSCode, open the Extensions view from Activity Bar, search `GitHub Copilot app modernization for Java` extension in marketplace. Select the Install button on the extension. After installation completes, you should see a notification in the bottom-right corner of VSCode confirming success.
 
 ## Migrate the Sample Java Application
 
-The following sections guide you through the process of migrating the sample Java application `asset-manager` to Azure using GitHub Copilot App Modernization for Java (Preview).
+The following sections guide you through the process of migrating the sample Java application `asset-manager` to Azure using GitHub Copilot app modernization for Java.
 
 ### Assess Your Java Application
 
 The first step is to assess the sample Java application `asset-manager`. The assessment provides insights into the application's readiness for migration to Azure.
 
 1. Open the VS code with all the prerequisites installed on the asset manager by changing the directory to the `asset-manager` directory and running `code .` in that directory.
-1. Open the extension `App Modernization for Java`.
-1. Hover the mouse over the **Assessment** section and click **Assess** button which looks like a triangle pointing right. Then, the Github Copilot Chat window will be opened and propose to run Modernization Assessor. Please confirm the tool usage by clicking **Continue**.
+1. Open the extension `GitHub Copilot app modernization for Java`.
+1. Hover the mouse over the **Assessment** view and click **Run Assessment** button which looks like a triangle pointing right. Then, the Github Copilot Chat panel will be opened and propose to run Modernization Assessor. Please confirm the tool usage by clicking **Continue**.
    
    ![Trigger Assessment](doc-media/trigger-assessment.png)
 
    > **NOTE**: If you are asked to allow the tool access the language models provided by GitHub Copilot Chat, select **Allow** to proceed.
 
 1. After each step, please manually input "continue" to confirm and proceed.
 1. Wait for the assessment to be completed and the report to be generated.
-1. Review the **Summary** report. Select **Propose Solution** to view the proposed solutions for the issues identified in the summary report.
-1. For this workshop, deselect all solutions and select **Migrate to Azure Database for PostgreSQL (SDK on Public Cloud)** in the Solution report, then select **Confirm Solution**.
+1. Review the **Assessment** report. Select the **Issues** tab to view the proposed solutions for the issues identified in the report.
```

Comment:
The recommended way would be click the "Migrate to Azure" button in the Quickstart section.